src/tools/genpolicy/src/utils.rs

```diff
@@ -16,11 +16,11 @@ struct CommandLineOptions {
     yaml_file: Option<String>,
 
     #[clap(
-        short,
         long,
-        help = "Optional Kubernetes config map YAML input file path"
+        num_args = 0..=std::usize::MAX,
+        help = "Optional Kubernetes YAML input file paths"
     )]
-    config_map_file: Option<String>,
+    data_source_files: Option<Vec<String>>,
```

Comment:
I wouldn't change this API as that would break downstreams.

Reply:
General thought: I was wondering what the compatibility guarantees of genpolicy are. The Cargo.toml we say `version = "0.1.0"`, but other components in kata also do this and are more or less version aligned with kata itself. At some point I'd like start a deprecation process to clean up flags which are more powerful / are the actual API we want/need to provide. 

In any case, I'll implement a flag migration then as outlined in the PR description